ansible_collections/arista/avd/roles/eos_designs/python_modules/ip_addressing/utils.py

```diff
@@ -107,3 +109,45 @@ def _mlag_odd_id_based_offset(self: "AvdIpAddressing") -> int:
             odd_id = self._mlag_secondary_id
 
         return int((odd_id - 1) / 2)
+
+    def _get_parallel_uplink_index(self: "AvdIpAddressing", uplink_switch_index: int) -> int:
```

Comment:
Let's add docstrings on these new functions.
Especially when there are multiple cases, it will help to describe it above.

Reply:
I added docstrings on these new utility functions and updated the docstrings in the p2p_uplink functions in avdipaddressing python_module, do let me know if any changes are necessary.